airbyte_cdk/sources/declarative/extractors/type_transformer.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import Any, Dict, Mapping
+
+
+@dataclass
+class AbstractTypeTransformer(ABC):
```

Comment:
Would we be able to rename this to `TypeTransformer`?

And in `record_selector.py`, we alias it to `DeclarativeTypeTransformer`, I think it would be a tiny developer experience improvement to not have this look like our other custom classes which aren't prefixed  by Abstract and the idea of Python abstract class interfaces.

```
from airbyte_cdk.sources.declarative.extractors.type_transformer import TypeTransformer as DeclarativeTypeTransformer
```

I guess the drawback is a user implementing the wrong interface, but how does this sound to you?

Reply:
I'm ok with this since we use direct import inside our CDK and expose only `TypeTransformer` from utils.